cardano-prelude.cabal

```diff
@@ -65,3 +65,52 @@ library
 
   if (!flag(development))
     ghc-options:         -Werror
+
+test-suite cardano-prelude-test
+  hs-source-dirs:      test
+  main-is:             test.hs
+  type:                exitcode-stdio-1.0
+
+  other-modules:
+                       Test.Cardano.Prelude
+                       Test.Cardano.Prelude.Base16
+                       Test.Cardano.Prelude.Gen
+                       Test.Cardano.Prelude.GHC.Heap
+                       Test.Cardano.Prelude.Golden
+                       Test.Cardano.Prelude.Helpers
+                       Test.Cardano.Prelude.Orphans
+                       Test.Cardano.Prelude.QuickCheck.Arbitrary
+                       Test.Cardano.Prelude.QuickCheck.Property
+                       Test.Cardano.Prelude.Tripping
+
+  build-depends:       base
+                     , aeson
+                     , aeson-pretty
+                     , attoparsec
+                     , base16-bytestring
+                     , bytestring
+                     , canonical-json
+                     , cardano-prelude
+                     , containers
+                     , cryptonite
+                     , formatting
+                     , ghc-heap
+                     , hedgehog
+                     , hspec
```

Comment:
Are using both `hspec` and `hedgehog` or was this just copy-pastaed ? Might be worth running `weeder` on this.

Reply:
`hspec` seems to be referenced from `Test.Cardano.Prelude.QuickCheck.Property`

Reply:
`weeder` doesn't report any redundant build-depends in the test suite.

Reply:
Fair enough!